docker/cygnus-ngsi/cygnus-entrypoint.sh

```diff
@@ -1072,8 +1072,8 @@ if [ "${CYGNUS_MULTIAGENT,,}" == "false" ]; then
     PIDS="$PIDS $!"
 fi
 
-touch /var/log/cygnus/cygnus.log && tail -f /var/log/cygnus/cygnus.log &
-PIDS="$PIDS $!"
```

Comment:
but why remove PIDS ?

Reply:
Fixed in [cf6daf6](https://github.com/telefonicaid/fiware-cygnus/pull/1887/commits/cf6daf6873ec36cf3e56c026d412619c9e856b91)